backend/project.clj

```diff
@@ -26,14 +26,15 @@
                  [org.apache.poi/poi-ooxml "3.8"]
                  [org.apache.tika/tika-core "1.6"]
                  [org.apache.tika/tika-parsers "1.6"]
-                 [org.slf4j/slf4j-api "1.7.7"]
-                 [org.slf4j/slf4j-simple "1.7.7"]
+                 [com.fzakaria/slf4j-timbre "0.3.8"]
+                 [org.slf4j/log4j-over-slf4j "1.7.25"]
+                 [org.slf4j/jul-to-slf4j "1.7.25"]
+                 [org.slf4j/jcl-over-slf4j "1.7.25"]
                  [com.google.gdata/core "1.47.1"]
                  [org.xerial/sqlite-jdbc "3.7.2"]
                  ; Akvo FLOW dependencies
                  [org.akvo.flow/akvo-flow "v1.9.29-52-g4bc8f24" :classifier "classes"]
```

Comment:
This will eventually be a tagged version right?

Reply:
Yes. There is a branch in Flow that once merged will make the tagged jars available for other projects.